### PR TITLE
transformers require sacremoses

### PR DESCRIPTION
**Pull request by roccqqck**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/sacremoses/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - sacremoses=sacremoses.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - click
```
Comment (chrisburr) [inline recipes/sacremoses/meta.yaml]: You probably don't need all of these, `python` and `pip` is likely enough for the `host` requirements.

File: recipes/transformers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - transformers=transformers.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - boto3
```
Comment (chrisburr) [inline recipes/transformers/meta.yaml]: You probably don't need all of these, `python` and `pip` is likely enough for the `host` requirements.

File: recipes/transformers/LICENSE [hunk 3]
```diff
@@ -0,0 +1,71 @@
+         Apache License
```
Comment (chrisburr) [inline recipes/transformers/LICENSE]: It looks like this file is included in the tarball on PyPI so you don't need to include it.

(The `licence_file` attribute is still needed)

File: recipes/transformers/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    - transformers.data.processors
+  commands:
+    - transformers --help
+  requires:
+    - pytest
+
+
+about:
+  home: https://github.com/huggingface/transformers
+  license: Apache Software
```
Comment (chrisburr) [inline recipes/transformers/meta.yaml]:
```suggestion
  license: Apache-2.0
```

File: recipes/sacremoses/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - sacremoses
+  commands:
+    - sacremoses --help
+
+about:
+  home: https://github.com/alvations/sacremoses
+  license: LGPL
```
Comment (chrisburr) [inline recipes/sacremoses/meta.yaml]:
```suggestion
  license: LGPL-2.1-or-later
```

File: recipes/transformers/LICENSE (hunk 3 above)
Comment (roccqqck) [reply recipes/transformers/LICENSE]: i removed the LICENSE file

but in meta.yaml

should i 
```
licence_file: LICENSE
```
or
```
licence_file: ""
```
????

Comment (chrisburr) [reply recipes/transformers/LICENSE]: It should be the filename of the file inside the tarball (which will almost certainly be the same as the name used on GitHub).

Comment (roccqqck) [reply recipes/transformers/LICENSE]: thank, i think i modified it like you suggested.

Comment (roccqqck) [reply recipes/transformers/LICENSE]: > It should be the filename of the file inside the tarball (which will almost certainly be the same as the name used on GitHub).

Sorry, I dont know where to ask, so I reply here.
These 2 packages haven't been uploaded yet.
https://github.com/conda-forge/sacremoses-feedstock
https://github.com/conda-forge/transformers-feedstock
Is there something wrong? 
How should I fix this?

Comment (chrisburr) [reply recipes/transformers/LICENSE]: They’ve been affected by https://github.com/conda-forge/conda-smithy/issues/1154

It’s been fixed now for new recipes but you’ll need to push a trivial or empty commit to trigger those two.

Comment (roccqqck) [reply recipes/transformers/LICENSE]: > They’ve been affected by [conda-forge/conda-smithy#1154](https://github.com/conda-forge/conda-smithy/issues/1154)
> 
> It’s been fixed now for new recipes but you’ll need to push a trivial or empty commit to trigger those two.

it worked!!! thank you!!!

**Pull request merged**